docs/reference/options.md

```diff
@@ -75,14 +75,19 @@ The configuration file uses JavaScript syntax, which means:
 | [`torznab`](#torznab)                               | **Required** |
 | [`torrentDir`](#torrentdir)                         | **Required** |
 | [`outputDir`](#outputdir)                           | **Required** |
+| [`dataDir`](#datadir)                               |              |
```

Comment:
```suggestion
| [`dataDirs`](#datadirs)                               |              |
```

Reply:
Yea, oversight, I think I did this all over, will fix it wide now.